src/Maker/MakeWebhook.php

```diff
@@ -0,0 +1,285 @@
+            RemoteEvent::class,
+            AbstractRequestParser::class,
+            RejectWebhookException::class,
+            RequestMatcherInterface::class,
+        ]);
+        $requestMatchers = [];
+        while (true) {
+            $newRequestMatcher = $this->askForNextRequestMatcher($io, $requestMatchers, $requestParserDetails->getFullName(), empty($requestMatchers));
+            if (null === $newRequestMatcher) {
+                break;
+            }
+            $requestMatchers[] = $newRequestMatcher;
+        }
+        $useChainRequestsMatcher = false;
+        // Use a ChainRequestMatcher if multiple matchers have been added OR if none (will be printed with an empty array)
+        if (1 !== \count($requestMatchers)) {
+            $useChainRequestsMatcher = true;
+            $useStatements->addUseStatement(ChainRequestMatcher::class);
+        }
+        $requestMatcherArguments = [];
+        foreach ($requestMatchers as $requestMatcherClass) {
+            $useStatements->addUseStatement($requestMatcherClass);
+            $requestMatcherArguments[$requestMatcherClass] = $this->getRequestMatcherArguments($requestMatcherClass);
+            if (ExpressionRequestMatcher::class === $requestMatcherClass) {
+                if (!class_exists(Expression::class)) {
+                    throw new \Exception('The ExpressionRequestMatcher requires the symfony/expression-language package.');
+                }
+                $useStatements->addUseStatement(Expression::class);
+                $useStatements->addUseStatement(ExpressionLanguage::class);
+            }
+        }
+
+        $this->generator->generateClass(
+            $requestParserDetails->getFullName(),
+            'webhook/RequestParser.tpl.php',
+            [
+                'use_statements' => $useStatements,
+                'use_chained_requests_matcher' => $useChainRequestsMatcher,
+                'request_matchers' => $requestMatchers,
+                'request_matcher_arguments' => $requestMatcherArguments,
+            ]
+        );
+    }
+
+    private function askForNextRequestMatcher(ConsoleStyle $io, array $addedMatchers, string $entityClass, bool $isFirstMatcher): ?string
+    {
+        $io->writeln('');
+        $availableMatchers = $this->getAvailableRequestMatchers();
+        $matcherName = null;
+        while (null === $matcherName) {
+            if ($isFirstMatcher) {
+                $questionText = 'Add a RequestMatcher (press <return> to skip this step)';
+            } else {
+                $questionText = 'Add another RequestMatcher? Enter the RequestMatcher name (or press <return> to stop adding matchers)';
+            }
+
+            $choices = array_diff($availableMatchers, $addedMatchers);
+            $question = new ChoiceQuestion($questionText, array_values(['<skip>'] + $choices), 0);
+            $matcherName = $io->askQuestion($question);
+            if ('<skip>' === $matcherName) {
+                return null;
```

Comment:
Do the same for other parts of this maker: Let's add so more spacing within the code. grouping multiple single line variable declarations is A-OK. But using that enter button between multi-line declarations, conditionals, etc... improves readability..

```suggestion
            $choices = array_diff($availableMatchers, $addedMatchers);
            $question = new ChoiceQuestion($questionText, array_values(['<skip>'] + $choices), 0);
            $matcherName = $io->askQuestion($question);
            
            if ('<skip>' === $matcherName) {
                return null;
```